docs/edgeql/ddl/functions.rst

```diff
@@ -45,8 +45,9 @@ CREATE FUNCTION
 
     # where <subcommand> is one of
 
-      FROM <language> <functionbody>
+      SET session_only := {true | false}
```

Comment:
```suggestion
      SET session_only := { true | false }
```

Reply:
The `{true | false}` is the current de facto standard in the docs.